openfl-workspace/torch_unet_kvasir/src/data_loader.py

```diff
@@ -121,6 +122,11 @@ def __init__(self, data_path, batch_size, **kwargs):
         """
         super().__init__(batch_size, **kwargs)
 
+        try:
+            int(data_path)
+        except:
+            raise ValueError("Expected `%s` to be representable as `int`.", data_path)
+
```

Comment:
By the way, I suppose you've re-tested all the workspaces modified in this PR, right?
If so, please add a "Testing done" section to the PR description.

Reply:
Had tested the change. It is available in the original description of the PR which has since been modified.